modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/objectViewContext.tsx

```diff
@@ -351,7 +378,10 @@ const viewReducer = (state: TState, action: TAction) => {
 					return {
 						...item,
 						defaultSort: defaultSortColumn ? true : false,
-						fieldLabel: item.label[defaultLanguageId],
+						fieldLabel: getLocalizableLabel(
+							item.label,
+							creationLanguageId
+						),
```

Comment:
Hey @alinedoleron, you can use the `item.name` as the `fallback` of the function.